site2/docs/reference-pulsar-admin.md

```diff
@@ -2267,35 +2230,28 @@ Options
 |`-l`, `--ledgerId`|The ledger id |0|
 |`-e`, `--entryId`|The entry id |0|
 
-### `get-backlog-quotas`
-Get the backlog quota policies for a topic.
+### `enable-deduplication`
+Enable a deduplication policy for a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics get-backlog-quotas tenant/namespace/topic
+$ pulsar-admin topics enable-deduplication tenant/namespace/topic
 ```
 
-### `set-backlog-quota`
-Set a backlog quota policy for a topic.
-
-Usage
-```bash
-$ pulsar-admin topics set-backlog-quota tenant/namespace/topic options
-=======
-### `get-deduplication`
-Get a deduplication policy for a topic.
+### `disable-deduplication`
```

Comment:
Same here

Reply:
@315157973 Just wanna double confirm 
![image](https://user-images.githubusercontent.com/53718687/107344814-2cef9d80-6ac3-11eb-84a4-5b99c99b1398.png)
all these commands are deprecated, right?

Reply:
The deprecated commands are in `initDeprecatedCommands` method of `CmdTopics.java`

Reply:
@315157973 @Jennifer88huang I just checked this java file, and I found all added commands in this pr are deprecated. Can you both help double confirm?

Reply:
![image](https://user-images.githubusercontent.com/9758905/107368716-dbbfc800-6b1b-11eb-8a2d-a7c8d7b118a5.png)


The deprecated command is in this method

Reply:
I am confused about the following commands (see https://github.com/apache/pulsar/pull/9108)
![image](https://user-images.githubusercontent.com/53718687/107370856-d050aa80-6ae3-11eb-8178-c958113a8e47.png)
Are `get-max-unacked-messages-on-subscription` and `get-max-unacked-messages-per-subscription` removed too? Or the second one is still in use?

Reply:
Based the code on the master branch, in the issue mentioned by @Jennifer88huang , the command names of topic-level and namespace-level have been unified, https://github.com/apache/pulsar/issues/9205

E.g:
The topic level `get-maxProducers` has been changed to `get-max-producers`, and `get-maxProducers` is marked as deprecated.

The deprecated command will not be deleted immediately, it can still be used, but it will no longer be displayed when user enters `--help`, and may be deleted at some point in the future.

Since the code is in the master branch, one thing that needs to be confirmed is whether this PR will be cherry-picked to 2.7.x and the document should correspond to it  @codelipenghui

Reply:
yes, I agree, I think we should apply the changes to 2.7.x.